src/pages/keto.tsx

```diff
@@ -59,6 +140,43 @@ const KetoPage = () => (
     />
 
     <Newsletter preselect={'keto'} />
+    <CompressedSection
+      expanded
+      left={
+        <>
+          <h3>Easy integration</h3>
+          <p>
+            {brandPrefix}Keto is a cloud native global access control server. It
+            is fully configurable, completely headless, and based on Google
+            Zanzibar. Take a global view of permissions management and scale on
```

Comment:
Are you talking about the reader or Keto here? If it is Keto,
```suggestion
            Zanzibar. Takes a global view of permission management and scales on
```

Reply:
@jaredpreston 👀  wdyt?

Reply:
from my understanding, this is correct @zepatrik - this text passage is speaking to the developer and is informing them how keto allows you take a global view of permissions and how it scales with your needs

Reply:
I believe the original to be correct, you "Take a global view of".